proto/spacemesh/v1/app_types.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+package spacemesh.v1;
+option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
+import "spacemesh/v1/types.proto";
+import "spacemesh/v1/var_types.proto";
+
+/////////////// Template API
+
+// The address of an app template in the templates global state store
+message TemplateAddress {
+    bytes address = 1;
+}
+
+// Template deploy data
+message DeployData {
+    TemplateAddress address = 1; // template address
+    // note - the author can be obtained from the deploy transaction but it is nice to have it here
+    AccountId author = 2; // template uploader
+    LayerNumber deploy_layer = 3; // layer that the template was deployed - 0 for genesis templates
```

Comment:
is this actually stored in the mesh data for a template? is it necessary for SVM? (both layer, and txid)